src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -399,6 +391,16 @@ def load(self,
         self.handler.module.set_localized_module_option(
             self.handler.MODULE_OPTION_NAME, None)
 
+    def __len__(self) -> int:
+        return len(self.schedules)
+
+    def load(self,
+             namespace_validator: Optional[Callable] = None,
+             image_validator: Optional[Callable] = None) -> None:
+        self.level_specs = {}
+        self.schedules = {}
```

Comment:
I'd add a line break here to make the three separate steps (1. reset 2. ingest global spec 3. ingest pool and image-level specs) clear.

Reply:
It looks like you have also added a extra line break:
```
        schedule_cfg = self.handler.module.get_module_option(
            self.handler.MODULE_OPTION_NAME, '')

        if schedule_cfg:
```
which isn't needed because these lines are part of the same "2. injest global spec" step ;)

Reply:
(I noticed only because I looked at the incremental diff instead of whole thing, feel free to ignore...)

Reply:
Thanks! Fixed.